defaults/initrd.scripts

```diff
@@ -1226,16 +1228,29 @@ write_env_file() {
 }
 
 crypt_filter() {
+	[ -z "${LUKS_KEY}" -o "$(echo ${LUKS_KEY} | grep -o '.gpg$')" = ".gpg" ]
+	local ask_pass=$?
+
 	if [ "${CRYPT_SILENT}" = '1' ]
 	then
 		eval run ${1}
+	elif [ "${PLYMOUTH}" = '1' ] && plymouth_running && [ ${ask_pass} -eq 0 ]
```

Comment:
BTW, why do you check every time if the plymouth is running? Is it really that unstable?

Reply:
I think you should move it near of `run plymouth` as in other cases:
`plymouth_running && run plymouth ...`